CONTRIBUTING.md

```diff
@@ -335,23 +353,19 @@ This is an authenticated local run that has the ability to run against a contain
 
 1. Confirm you've installed all recommended tooling
 1. Confirm the repository name has no blank spaces in it. If it does replace that blank with a dash or underscore, the container tooling may have issues with unescaped parameter strings.
-1. Confirm you've installed resources through yarn
-1. Create a local dotenv file called `.env.local` in the root of Quipucords-UI, and add the following contents
-    ```
-    REACT_APP_DEBUG_MIDDLEWARE=true
-    ```
+1. Confirm you've installed resources through npm
 1. Make sure podman desktop is running
 1. Open a couple of instances of Terminal and run...
     ```
-    $ yarn start:stage
+    $ npm run start:stage
```

Comment:
@cdcabrera I am not sure if/how we should document the npm run start:stage. We can't really run scans anymore with this command, since we moved the server to celery workers. Maybe this command should come with a warning ?

Reply:
These notes are meant to aid in a quick-start process. I wouldn't go beyond a generic response of something like
>  "[there will be limitations to running this emulated API. check with the team to understand current limitations]"

...since even the missing ability to scan could shift again and you'd have to update this again.